src/stratigraphy/util/description_block_splitter.py

```diff
@@ -128,25 +126,16 @@ def separator_condition(self, last_line: TextLine, current_line: TextLine, curre
         for line in self.geometric_lines:
             line_y_coordinate = (line.start.y + line.end.y) / 2
 
-            line_cuts_lefthandside_of_block = line.start.x < current_block.rect.x0 < line.end.x
+            line_cuts_lefthandside_of_text = (line.start.x < last_line.rect.x0 < line.end.x) and (
+                line.start.x < current_line.rect.x0 < line.end.x
+            )
```

Comment:
Basically, you are saying that the indicator line should cut in between two lines. Therefore we look at where those two lines start. But it is not so important where the full block starts. Is that correct?

I don't see any strong reason for one or the other. As long as the scores improve or remain the same.

I imagine there is some difference between block.rect.x0 and last_line.rect.x0.

Reply:
Yes, correct. The difference is small, though maybe using the more "local" context is slightly more reliable, as we prevent a single incorrectly identified text line further away from potentially influencing the condition.